app/src/main/res/layout/unanswered_surveys_header_view.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<TextView
+  android:id="@+id/heading"
+  style="@style/SubheadPrimary"
+  xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:tools="http://schemas.android.com/tools"
```

Comment:
just want to clarify too that it's style for consistency throughout our code base, not just a personal preference of mine :)